src/rust/generators/sysmon-generator/src/models/mod.rs

```diff
@@ -114,21 +103,19 @@ fn strip_file_zone_identifier(path: &str) -> &str {
 }
 
 /// Gets the name of the process given a path to the executable.
-fn get_image_name(image_path: &str) -> Option<String> {
+fn get_image_name(image_path: &str) -> String {
```

Comment:
I don't even really like that we're doing this, I think we should drop this and pass up the full path, especially because it's Windows specific, but the reason why I changed the return type is because the generator had calls to `.unwrap()` after calling this, which was not acceptable.